lib/northbound.c

```diff
@@ -787,6 +797,12 @@ void nb_candidate_edit_config_changes(struct nb_config *candidate_config,
 			*error = true;
 			continue;
 		}
+		if (!operation_valid(nb_node, change))
```

Comment:
Can you explain what exactly was the issue that you tried solving with this new function?